### PR TITLE
[FIX] Workaround for use of the default settings collection

### PR DESCRIPTION
**Pull request by MartinSchoeler**
This is a workaround for us to swiftly be able to use the default settings collection and system.

This will allow us to use the default settings ui for now, after the MVP we will discuss again a better way to maintain traceability.

### CODE REVIEW

**Review by ggazzo — CHANGES_REQUESTED** [review]
use return first

**Review by tiagoevanp — APPROVED** [review]

File: server/services/voip/service.ts [hunk 1]
```diff
@@ -84,7 +85,34 @@ export class VoipService extends ServiceClass implements IVoipService {
...
+							password: settings.get('VoIP_Management_Server_Password'),
+						},
+				  }
+				: {
+						configData: {
+							websocketPort: parseInt(settings.get('VoIP_Server_Websocket_Port')),
+							websocketPath: settings.get('VoIP_Server_Websocket_Path'),
+						},
+				  }),
+		} as IVoipServerConfig;
```
Comment (sampaiodiego) [inline server/services/voip/service.ts:115]: looks like the type casting is not needed.
```suggestion
		};
```

File: server/services/voip/service.ts [hunk 2]
```diff
@@ -84,7 +85,34 @@ export class VoipService extends ServiceClass implements IVoipService {
...
 	async getServerConfigData(type: ServerType): Promise<IVoipServerConfig | null> {
-		return this.VoipServerConfiguration.findOne({ type, active: true });
+		// TODO: Decide the approach we will take regarding settings after the MVP,
+		// For now this work around should be enough.
+
+		// const config = this.VoipServerConfiguration.findOne({ type, active: true });
+
+		const management = type === ServerType.MANAGEMENT;
+		return {
+			_id: '',
```
Comment (sampaiodiego) [inline server/services/voip/service.ts:95]: looks like `IVoipServerConfig` doesn't need extend `IRocketChatRecord` anymore.. so you don't need to provide `_id: ''` (which doesn't make sense anyways)

File: server/services/voip/service.ts [hunk 3]
```diff
@@ -84,7 +85,34 @@ export class VoipService extends ServiceClass implements IVoipService {
...
+			type,
+			host: settings.get(management ? 'VoIP_Management_Server_Host' : 'VoIP_Server_Host'),
+			name: settings.get(management ? 'VoIP_Management_Server_Name' : 'VoIP_Server_Name'),
+			active: true,
+			_updatedAt: new Date(),
```
Comment (sampaiodiego) [inline server/services/voip/service.ts:100]: same as before, not extending `IRocketChatRecord` this is not required

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

**Review by sampaiodiego — APPROVED** [review]
oh, not sure how my suggestions didn't break other parts of the code.. 🙊  I see `IVoipServerConfig` still being used as a type for a database record, like on this model: 
https://github.com/RocketChat/Rocket.Chat/blob/780dabee9d3d15b2b30876a6d6adadf76f52f628/app/models/server/raw/VoipServerConfiguration.ts#L4

I'll approve this PR anyways as it seems that there still work needed to make this "voip server config" thing to work, and I think it can be done on the target branch.

**Pull request merged**